SnowGlobe/main.py

```diff
@@ -53,49 +54,64 @@ def play_song(song_number):
     eighth_note = whole_note / 8
 
     # set up note values
-    A3 = 220
-    Bb3 = 233
-    B3 = 247
+    # A3 = 220
```

Comment:
Why are these commented too

Reply:
pylint was complaining that the variables weren't being used. I can uncomment them and add a ignore rule to pylint so that it's fine with unused variables.

Reply:
Ah ok. An ignore rule would be awesome. Thanks!

Reply:
I've removed the comments for all of these